### ISSUE TITLE
Inconsistent tildification in recently opened

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#120510]
![image](https://user-images.githubusercontent.com/323878/113515241-295b1e80-9528-11eb-8974-6fd542bf3b02.png)

I think these should all have tilde

**Comment by isidorn** [comment]
Makes sense. I am looking for an owner of the label service since I have no cycles to make updates to it at the moment.

**Comment by JacksonKearl** [comment]
Do you still see this? I cleared my recently opened a while back and haven't seen any since. I wonder if you may have some old style entries cached?

**Comment by roblourens** [comment]
I still have them, but I will delete them and see if they come back

**Comment by bpasero** [comment]
This is fixed.
